program/Project Descriptions and Plans/CV32E40Pv2/Project_Concept_for_CV32E40Pv2_June 28.md

```diff
@@ -0,0 +1,202 @@
+# OpenHW Project Concept Proposal: CV32E40Pv2
+*The PC proposal introduces the project and explains the market drivers and the "why"* 
+
+# Title of Project - "CV32E40Pv2"
+# Project Concept Proposal
+## Date of proposal - 2021-06-28
+## Author(s) - Olivier Montfort (Dolphin Design), Pascal Gouedo (Dolphin Design), Yoann Pruvost (Dolphin Design)
+
+## High Level Summary of project, project components, and deliverables
+This proposal is about verifying PULP basic instructions disabled during CV32E40P v1 phase improving performances, power consumption and even code size depending of application programs.

```

Comment:
This proposal is about verifying XPULP, RVF, and Zfinx ISA extensions that were disabled [..]

Reply:
updated